### PR TITLE
dev/core#4696 - Increase length of ActionSchedule.name and fix upgrade error

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
Fixes [dev/core#4696](https://lab.civicrm.org/dev/core/-/issues/4696)

Technical Details
----------------------------------------
Fixes upgrade crash by increasing the column length before writing to it.
Also fixes an error in the upgrader where `Discount.entity_table` and `ActionSchedule.name` descriptions were accidentally swapped.

### CODE REVIEW

**Comment by totten** [comment]
Test failure was because it needed version-bump. For stable branch, it should be fine to do the `set-version` within the PR.

File: CRM/Upgrade/Incremental/php/FiveSixtySix.php [hunk 1]
```diff
@@ -83,6 +84,17 @@ public function upgrade_5_66_beta1($rev): void {
...
   }
 
+  /**
+   * Upgrade step; adds tasks including 'runSql'.
+   *
+   * @param string $rev
+   *   The version number matching this function name
+   */
+  public function upgrade_5_66_1($rev): void {
+    $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
```
Comment (totten) [inline CRM/Upgrade/Incremental/php/FiveSixtySix.php:94]: Test failure relates.

FWIW, if you want a a quicky-generic way to run upgrade-test on a local build, there's this command:

```bash
civibuild ut dmaster 5.45*
```

Comment (colemanw) [reply CRM/Upgrade/Incremental/php/FiveSixtySix.php]: Thanks... I just remembered `set-version` doesn't generate that sql file for stable releases.

**Comment by totten** [comment]
Tested and confirmed the upgrade-scenario passes

**Pull request merged**